roborazzi-compose-preview-scanner-support/src/main/java/com/github/takahirom/roborazzi/RoborazziPreviewScannerSupport.kt

```diff
@@ -13,49 +13,52 @@ import sergio.sastre.composable.preview.scanner.core.preview.ComposablePreview
 fun ComposablePreview<AndroidPreviewInfo>.captureRoboImage(
   filePath: String,
   roborazziOptions: RoborazziOptions = provideRoborazziContext().options,
-  configBuilder: RoborazziComposeConfigBuilder = this.toRoborazziComposeConfigBuilder()
+  roborazziComposeOptions: RoborazziComposeOptions = this.toRoborazziComposeOptions()
```

Comment:
@sergio-sastre
Thanks. I was almost ready to release this, but while writing the release notes, I realized it might be a bit strange to pass a builder to the APIs. Additionally, the names `RoborazziComposeConfig` and `RoborazziComposeConfigBuilder` felt odd because `RoborazziComposeConfig` is not actually created by `RoborazziComposeConfigBuilder`. 

I also think we don’t have a strong preference for using `Config` over `Options`, especially since we’re already using `RoborazziOptions`. Therefore, I renamed it to `RoborazziComposeOptions`.

I made some adjustments. Could you take a look?

Reply:
@sergio-sastre This is going to block other tasks, so I'll merge this now. If you have any concerns, let me know. I think we can address them in a patch release.